fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -2491,15 +2491,20 @@ private List<List<String>> doPredicate(ShowStmt showStmt,
         return returnRows;
     }
 
-    private void handleShowCreateExternalCatalog() {
+    private void handleShowCreateExternalCatalog() throws AnalysisException {
```

Comment:
Better to add a sql-test case to cover this case.

Reply:
+1

Reply:
done